pkg/diagnostics/cluster/aggregated_logging/daemonsets.go

```diff
@@ -0,0 +1,109 @@
+package aggregated_logging
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapisext "k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+const daemonSetNoLabeledNodes = `
+There are no nodes that match the selector for DaemonSet '%[1]s'.  
+An example of a command to target a specific node for this DaemonSet:
+
+   oc label node/ip-172-18-2-170.ec2.internal %[2]s
+
+or to label them all:
+
+   oc label node --all %[2]s
+`
+
+const daemonSetPartialNodesLabeled = `
+There are some nodes that do not match the selector for DaemonSet '%s'.  
+A list of these nodes can be discovered by running:
```

Comment:
wouldn't this be the list of nodes that do match, might be more clear if this sentence said "A list of matching nodes can be discovered by running:"